woodwork/utils.py

```diff
@@ -291,12 +291,23 @@ def _is_valid_latlong_value(val, bracket_type=tuple):
     return False
 
 
-def _is_null_latlong(val):
-    if isinstance(val, str):
-        return val == "None" or val == "nan" or val == "NaN"
+def _is_nan(value):
+    """This function checks if string values are common NaN values.
+    Lists are not counted as NaN, and all other values are passed to pd.isnull
+    """
+    if isinstance(value, str):
+        return value in ww.config.get_option("nan_values")
+    if isinstance(value, list):
+        return False
+    return pd.isnull(value)
+
+
+def _is_latlong_nan(value):
+    """Checks if a LatLong value is NaN"""
+    if isinstance(value, tuple):
```

Comment:
Koala stores the latlongs as lists, because it can't handle tuples. You may need to include `list` here as well--the fact that it's not currently failing makes me wonder if I'm missing something here.

Reply:
@tamargrey it falls through and gets fixed down here: https://github.com/alteryx/woodwork/pull/1255/files#diff-05318c9c3bf0f63e471074de1ccab68b2c818c14ba9ae3a0220e510264467d44L248

Reply:
I was more thinking that something like `[np.nan, np.nan]` would get passed in and not get recognized as needing to have its individual values checked so `_is_nan([np.nan, np.nan])` will get called and return False, which won't be doing the check we want. 

in the latlong reformatting, this doesn't end up making a difference, because no transformation was needed, but in `describe`, wouldn't it not get counted as a nan?

Reply:
Ah, I see what you mean. Will fix